plugins/toolbox/src/components/Root/tools.tsx

```diff
@@ -114,6 +115,12 @@ export const defaultTools: Tool[] = [
     component: <TimeConverter />,
     category: 'Convert',
   },
+  {
+    id: 'sla-calculator',
+    name: 'SLA Calculator',
```

Comment:
Could you use small case for the Calculator like other tools are using for 2nd word? So "SLA calculator"

Reply:
Done.